domenetjenester/beregning-ytelse/pom.xml

```diff
@@ -59,6 +59,27 @@
 			<artifactId>behandlingslager-testutilities</artifactId>
 			<scope>test</scope>
 		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>no.nav.k9.felles.integrasjon</groupId>
+            <artifactId>k9-pdl-klient</artifactId>
```

Comment:
Denne er for å hente antall barn for beregning av barnetillegg?

Reply:
Ja, men koden er flyttet så denne kan fjernes